src/games/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { Game } from '@/store/types'
+
+export const getGameName = (game: Game): string => {
```

Comment:
Please rename const. Return isn't name of game.

Also if no matches are made, would better to throw error. When error is triggered, it gets logged into sentry at right point instead the flow going further and crashing later or sooner into another error.

Reply:
Fixed